src/settings/Translations.js

```diff
@@ -3,11 +3,45 @@ import PropTypes from 'prop-types';
 import { FormattedMessage } from 'react-intl';
 import { Pane } from '@folio/stripes/components';
 
+function renderArray(a) {
+  return a.map(t => {
+    let str;
+    switch (t.type) {
+      case 1: // placeholders for strings, e.g. {foo}
+      case 2: // placeholders with types, e.g. {foo, number}
+        str = `{${t.value}}`;
+        break;
+      case 8: // html elements, e.g. <strong>foo</strong>
```

Comment:
Yikes! It's always scary to see a constant like `8` in code. But I trust your judgement on this.